lib/exq/manager/server.ex

```diff
@@ -109,6 +109,21 @@ defmodule Exq.Manager.Server do
     Enqueuer.enqueue_in(state.enqueuer, from, queue, offset, worker, args)
     {:noreply, state, 10}
   end
+  
+  def handle_call({:subscribe, queue}, from, state) do
+    updated_state = add_queue(state, queue)
+    {:reply, :ok, updated_state}
```

Comment:
It's using ```{reply,Reply,NewState,Timeout}``` response for GenServer handle_call to set the next timeout call (see http://www.erlang.org/doc/man/gen_server.html).

The other option would have been to use a separate timeout call, but at the time this was more flexible.